packages/jsii/test/negatives/neg.submodules-cannot-share-symbols.ts

```diff
@@ -1,4 +1,4 @@
-///!MATCH_ERROR: Symbol is re-exported under two distinct submodules (ns1 and ns2)
+///!MATCH_ERROR: Symbol is re-exported under two distinct submodules
```

Comment:
Why this change?

Reply:
Because on my machine this was breaking the build. The error message I got said `(ns2 and ns1)` (order different way around), and I did not want to fight with apparent nondeterminism.

Reply:
Ohhhh yeah makes sense!